eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/AbstractHTTPServer.java

```diff
@@ -366,9 +367,12 @@ public void channelRead(final ChannelHandlerContext ctx, final Object message) {
 
                 if (useRequestURI) {
                     if (useTrace) {
-                        span.setAttribute(SemanticAttributes.HTTP_METHOD, httpRequest.method().name());
-                        span.setAttribute(SemanticAttributes.HTTP_FLAVOR, httpRequest.protocolVersion().protocolName());
-                        span.setAttribute(SemanticAttributes.HTTP_URL, httpRequest.uri());
+                        span.setAttribute(SemanticAttributes.HTTP_METHOD,
+                                Objects.requireNonNull(Objects.requireNonNull(httpRequest).method()).name());
+                        span.setAttribute(SemanticAttributes.HTTP_FLAVOR,
```

Comment:
I wonder if is it possible that the request is null or the request's method is null?

Reply:
@horoc 
add npe check
        public void channelRead(final ChannelHandlerContext ctx, final Object message) {
            final HttpRequest httpRequest = (HttpRequest) message;

            if (httpRequest == null) {
                return;
            }